utils/create-news-verification-hashes/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * SCRIPT FOR NEWS FEED VERIFICATION HASH CREATION
+ *
+ * COMMANDS:
+ * 1. Fetch ALL files and create hashes:
+ *   yarn create-news-verification-hashes
+ *
+ * 2. Fetch file for specific environment and create hash:
+ *   NEWS_ENV=development yarn create-news-verification-hashes
+ */
+
+const lodash = require('lodash');
+const https = require('https');
+const crypto = require('crypto');
+const fs = require('fs');
+
+const readModuleFile = (path, callback) => {
+  try {
+    const filename = require.resolve(path);
+    fs.readFile(filename, 'utf8', callback);
+  } catch (e) {
+    callback(e);
+  }
+}
+
+// Start script
+console.log('\n\x1b[36m%s\x1b[0m', 'Creating news verification hashes...\n');
+
+const newsEnvironment = process.env.NEWS_ENV;
+const allowedFiles = [
+  { name: 'newsfeed_development.json', env: 'development' },
+  { name: 'newsfeed_mainnet.json', env: 'mainnet' },
+  { name: 'newsfeed_staging.json', env: 'staging' },
+  { name: 'newsfeed_testnet.json', env: 'testnet' },
+  { name: 'news.dummy.json', env: 'dummy_development' }, // Faked test file for development purposes
+];
+
+let filesToHash = [];
+if (newsEnvironment) {
+  const fileName = `newsfeed_${newsEnvironment.toLowerCase()}.json`;
+  const fileAllowed = lodash.find(allowedFiles, (allowedFiles => allowedFiles.name === fileName));
```

Comment:
Better call `(allowedFile => allowedFile.name...`, as `allowedFiles` is already defined in the upper scope.

Reply:
@daniloprates typo, nice catch 🚀